Sources/AppStoreConnectCLI/Commands/TestFlight/Apps/ReadAppCommand.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Foundation
+
+struct ReadAppCommand: CommonParsableCommand {
+
+    static var configuration = CommandConfiguration(
+        commandName: "read",
+        abstract: "Find and read app info"
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(
+        help: ArgumentHelp(
+            "Filter by app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
+            discussion: "Please input either app id or bundle Id",
+            valueName: "app-id / bundle-id"
+        )
+    ) var appIdOrBundleId: String
```

Comment:
We can further improve this by transforming the input as part of the argument definition:
```suggestion
    @Argument(
        transform: CommandArgument.init
        help: ArgumentHelp(
            "Filter by app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
            discussion: "Please input either app id or bundle Id",
            valueName: "app-id / bundle-id"
        )
    ) var appIdOrBundleId: CommandArgument
```